Xamarin.Forms.Build.Tasks/CompiledMarkupExtensions/StaticExtension.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+using Xamarin.Forms.Xaml;
+using System.Xml;
+
+using static System.String;
+
+namespace Xamarin.Forms.Build.Tasks
+{
+	class StaticExtension : ICompiledMarkupExtension
```

Comment:
Remember the compiledConverters branch? This is the same for markups. Except it's only applied (for now) on StaticExtension and there are no extensions points (using Attributes)